### ISSUE TITLE
Clear up error messages

### ISSUE THREAD

**Issue by GammaSQ** [QubesOS/qubes-issues#4532]
### Qubes OS version: 4.0

A lot of commands fail with exceptions instead of clearly understood error messages. I guess this is common among all qvm-commands, I simply didn't test them all for every error-case. So here are just three examples:

    qvm-run [VM] [CMD]
    [Ctrl+C]
    Traceback: [python-error-message...]
    KeyboardInterrupt

Especially confusing: running command in any VM and interrupting doesn't change the running command in the VM. (so e.g. firefox will continue running)  
But interrupting a command run in a dispVM (`qvm-run --dispvm -- firefox` + Ctrl+C) will kill the dispVM.
(killing at the wrong time will keep the commandline red.)

---------

    qvm-pci d [VM] [VM:nonexisting device]
    Got empty response from qubesdb. See journalctl in dom0 for details.

---------

    qvm-copy [already copied file to same VM]
    A file named [filename] already exists in QubesIncoming dir
    (zenity:13320): Gtk-WARNING ** : Theme parsing error: [... several lines of similar Gtk-warnings]

This one might be xfce-specific.

**Comment by GammaSQ** [comment]
> The first tells you to check the logs, and `journalctl` is sufficiently verbose.

What I don't see is why I have to check there at all? Sanity checking these commands should be easy. In this specific case, I imagine qubesdb is queried with VM and device-ID? Then an empty response could be translated to the user as: "Device [device-ID] does not exist in [VM]!"

**Comment by marmarek** [comment]
> Especially confusing: running command in any VM and interrupting doesn't change the running command in the VM. (so e.g. firefox will continue running)

Right now qrexec protocol does not support "interrupt" signal of any kind, which is intentional in most cases. qrexec is meant as a transport layer between domains potentially with different trust levels. Interrupting a service in wrong time may have serious implications, including security issues.
`qubes.VMShell` service, especially when called from dom0, is somehow special case here, because
1. most shell command are expecting to be interrupted with SIGINT/Ctrl+C if necessary
2. this service already gives the ultimate control over its target, so nothing worse can happen

Maybe the way forward would be a per-service configuration if given service call can be interrupted - by default enabled only for `qubes.VMShell`. But this is not a simple change, involve protocol change etc.

**Comment by GammaSQ** [comment]
Just for the record, I'm not sure quitting the command running in the VM is the best way to go. But the current error-message leaves the impression something went wrong which, as far as I can tell, isn't the case.

**Comment by andrewdavidwong** [comment]
> Bigger picture, a content-audit of all user-facing messages with recommended edits based on plain language and user-centric best practices, should happen. Not sure if an Issue has already been created for this, but flagging here.

I agree, @ninavizz, and I'm not aware of an existing issue for this. Please feel free to create one.

**Comment by ninavizz** [comment]
Updated 2018 milestone from `4.1` to `ongoing` as there's no linked PRs

**Comment by andrewdavidwong** [comment]
> Updated 2018 milestone from `4.1` to `ongoing` as there's no linked PRs (unless @marmarek's 2018 PR fixed the specific nit this issue was created to correct?)

Good thought to change the milestone from 4.1, but it should be "TBD," not "Ongoing." "Ongoing" is for things that are actually ongoing and are not (and will not) be implemented and released with any particular Qubes OS release. For example, a lot of documentation issues are on the "Ongoing" milestone because they are not release-specific.

**Comment by parulin** [comment]
Tools that should handle `Ctrl+C` gracefully:

* `qvm-template`
* `qvm-template-gui`

**Comment by ben-grande** [comment]
> qvm-pci d [VM] [VM:nonexisting device]
> Got empty response from qubesdb. See journalctl in dom0 for details.

Tested wit hhttps://github.com/QubesOS/qubes-core-admin/pull/751 and still the case. Still needs to send appropriate exceptions from `qubes.ext.pci`.
